MediaPlayer.py

```diff
@@ -0,0 +1,55 @@
+import re
+from pygame import mixer
+import os
+
+
+# p = "C:\Users\Mulat\Desktop\MediaPlayerProject\songs"
+
+
+def load_song_dir_to_dictionary() -> dict:
+    user_path = input()
+    songs_name = os.listdir(user_path)
+    files_path = [os.path.abspath(user_path) for x in os.listdir(user_path)]
```

Comment:
using an indexed loop in python is usually not the best option, and creating your list and then changing its elements is also less clean than creating the data you want right away. Try switching out lines 12 - 17 with a single statement that returns a dictionary built using a dict comprehension. If it feels crowded it can usually be refactored a bit later.